.github/workflows/percy.yml

```diff
@@ -25,36 +25,12 @@ jobs:
       - name: Download OTP2 config file
         run: curl $PERCY_OTP2_CONFIG_URL --output /tmp/otp2config.yml
         env:
-          PERCY_OTP2_CONFIG_URL: ${{ secrets.PERCY_OTP2_CONFIG_URL_METRO_MODE_SELECTOR }}
+          PERCY_OTP2_CONFIG_URL: ${{ secrets.PERCY_MOCKOTP2_CONFIG_URL_METRO_MODE_SELECTOR }}
       - name: Take Percy Snapshots
         # Move everything from latest commit back to staged
         run: npx percy exec -- npx jest percy/percy.test.js --force-exit
         env:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_OTP2 }}
           PERCY_OTP_CONFIG_OVERRIDE: /tmp/otp2config.yml
           OTP_RR_PERCY_MOBILE: true
-  run-pixel-tests-with-otp2-real-server-call-taker:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          # This allows us to work with the repository during the lint step
-          fetch-depth: 2
-      - name: Use Node.js 16.x
-        uses: actions/setup-node@v1
-        with:
-          node-version: 16.x
-      - name: Install npm packages using cache
-        uses: bahmutov/npm-install@v1
-      - name: Download OTP2 config file
-        run: curl $PERCY_OTP2_CONFIG_URL --output /tmp/otp2config.yml
-        env:
-          PERCY_OTP2_CONFIG_URL: ${{ secrets.PERCY_OTP2_CONFIG_URL_METRO_MODE_SELECTOR }}
-      - name: Take Percy Snapshots
-        # Move everything from latest commit back to staged
-        run: npx percy exec -- npx jest percy/percy.test.js --force-exit
-        env:
-          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_CALL_TAKER_OTP2 }}
-          PERCY_OTP_CONFIG_OVERRIDE: /tmp/otp2config.yml
-          OTP_RR_PERCY_CALL_TAKER: true
+  # TODO: Reinstate calltaker UI tests.
```

Comment:
I'm ok with re-adding the call taker tests separately, but ti seems like the tests are now running in mobile mode only (at least according to the Percy results)

Reply:
Call-taker is back with https://github.com/opentripplanner/otp-react-redux/pull/972/commits/8b388e3b41bc342c9cf1774c91b44ca1324ddfbb (although only one screenshot is taken of the main panel, the other steps are the same as desktop so they are skipped).

Reply:
I'm not sure if we can just skip them -- we need the screenshots to make sure that there are no regressions in the trip planning form or the batch itinerary

Reply:
The first Calltaker screenshot occurs after a trip query is made (but it is the same query result as the regular config). 

How about letting the Calltaker test run without taking screenshots? 
The other steps after that are identical to the ones for regular desktop and would result in duplicate screenshots.